docs/backend-code-convention.md

```
@@ -0,0 +1,1148 @@
+
+```
+
+- try 블록은 `실패할 수 있는 단 하나의 행위`만 감싸야 한다.
+    - 실패할 가능성이 있는 핵심 비즈니스 로직 단위로 최소화해야 한다.
+    - try-catch 안에 단 한 줄의 코드만 있을 수 있도록 노력한다.
+
+# 4. 테스트
+
+- 테스트 파일은 반드시 테스트 대상 클래스와 동일한 패키지에 위치해야 한다.
+- 단위 테스트가 아니라면 `@SpringBootTest` , `@WebMvcTest`를 사용한다.
+    - `@SpringBootTest` : 통합 테스트
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)